src/main/java/com/truelayer/java/payments/entities/retry/SmartRetry.java

```diff
@@ -0,0 +1,18 @@
+package com.truelayer.java.payments.entities.retry;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@Builder
+@ToString
```

Comment:
This `@ToString` is use to improve test readability :
![Screenshot 2022-12-14 at 18 49 34](https://user-images.githubusercontent.com/10221348/207669943-186ecb94-2677-43be-bd38-4e181f1b3229.png)